example/zend-3.0.3/ZEND_README.md

```diff
@@ -0,0 +1,223 @@
+# ZendSkeletonApplication
+
+## Introduction
+
+This is a skeleton application using the Zend Framework MVC layer and module
+systems. This application is meant to be used as a starting place for those
+looking to get their feet wet with Zend Framework.
+
+## Installation using Composer
+
+The easiest way to create a new Zend Framework project is to use
+[Composer](https://getcomposer.org/).  If you don't have it already installed,
+then please install as per the [documentation](https://getcomposer.org/doc/00-intro.md).
+
+To create your new Zend Framework project:
+
+```bash
+$ composer create-project -sdev zendframework/skeleton-application path/to/install
+```
+
+Once installed, you can test it out immediately using PHP's built-in web server:
+
+```bash
+$ cd path/to/install
+$ php -S 0.0.0.0:8080 -t public/ public/index.php
+# OR use the composer alias:
+$ composer run --timeout 0 serve
```

Comment:
Is this a better way of running the example rather than having to download and use MAMP?